Hearthstone Deck Tracker/Importing/CardImageImporter.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Hearthstone_Deck_Tracker.Hearthstone.Entities;
+using Hearthstone_Deck_Tracker.Utility.Logging;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hearthstone_Deck_Tracker.Importing
+{
+	public static class CardImageImporter
```

Comment:
Similar to the namespace, this is not really an "importer" in the sense of other importers HDT has. 

`AssetDownloader`, `ArtAssets` or just `Assets` might be more fitting.